.gitlab-ci.yml

```diff
@@ -5,7 +5,7 @@ stages:
 
 variables:
   PYVERSION: "3.8"
-  RSMVERSION: "11.0.1"
+  RSMVERSION: "11.1.1"
```

Comment:
Did only the middle number change?

Reply:
Yes, we had 11.1.0 but there was a problem with it and we can't reuse version numbers on PyPI so we had to go to 11.1.1.

Reply:
11.1.0 failed, so I fixed (hopefully) and increased version.